docs/data/release_archive.yml

```diff
@@ -428,6 +428,11 @@ release_archive:
       version: 1.0.0
       release_date: 2022-12-21
       filename: "opensearch"
+    - name: "Flink MongoDB Connector"
```

Comment:
Consider adding version 1.0.0 which seems to not have been listed in the last release.

Reply:
+1